CitationStyleLanguageSettingsForm.inc.php

```diff
@@ -91,9 +91,9 @@ public function fetch($request, $template = null, $display = false) {
 		$templateMgr = TemplateManager::getManager($request);
 		$templateMgr->assign(array(
 			'pluginName' => $this->plugin->getName(),
-			'primaryCitationStyleListData' => json_encode($primaryCitationStyleList->getConfig()),
-			'citationStylesListData' => json_encode($citationStylesList->getConfig()),
-			'citationDownloadsListData' => json_encode($citationDownloadsList->getConfig()),
+			'primaryCitationStyleListData' => $primaryCitationStyleList->getConfig(),
+			'citationStylesListData' => $citationStylesList->getConfig(),
+			'citationDownloadsListData' => $citationDownloadsList->getConfig(),
```

Comment:
(Happy to see this stuff get cleaned up!)

Reply:
OTOH, the only problem I encountered when doing this is null values; I think `json_encode` converts them into empty strings rather than `null` or `''` or `0` or whatever. Have you encountered that? (IIRC it had me using `...|json_encode|default:0` in places in the templates.)

Reply:
I have not run into this myself. Does it happen for all `null` values inside of an assoc array? Like this:

```
$data = [
  'test' => null,
];
```

Or only when encoding a null variable directly, like `json_encode(null)`? 

If the issue only occurs with the latter, I don't think that should be a problem for us. The data that is passed into a UI component should _never_ be just `null`.